### PR TITLE
JW-1113 fixed nil pointer dereferences

### PR DESCRIPTION
**Pull request by mgreg0r**

### CODE REVIEW

**Review by m-kostrzewa — APPROVED** [review]

**Comment by m-kostrzewa** [comment]
@mgreg0r I added @PawelMarchewkaCodiLime as reviewer to familiarize him with the codebase

File: driver/driver.go [hunk 1]
```diff
@@ -349,11 +349,11 @@ func (d *ContrailDriver) CreateEndpoint(req *network.CreateEndpointRequest) (
 	}
 
 	contrailIP, err := d.controller.GetOrCreateInstanceIp(contrailNetwork, contrailVif)
-	instanceIP := contrailIP.GetInstanceIpAddress()
-	log.Infoln("Retreived instance IP:", instanceIP)
 	if err != nil {
 		return nil, err
 	}
+	instanceIP := contrailIP.GetInstanceIpAddress()
+	log.Infoln("Retreived instance IP:", instanceIP)
```
Comment (kaweue) [inline driver/driver.go]: Retreived -> Retrieved

**Review by kaweue — CHANGES_REQUESTED** [review]

**Review by kaweue — APPROVED** [review]

**Review by m-kostrzewa — APPROVED** [review]
:roll_eyes:

File: driver/driver.go (hunk 1 above)
Comment (m-kostrzewa) [reply driver/driver.go]: I now know why I made this typo in the first place: it's because I confused it with spelling of `receive`.

**Pull request merged**